tests/Areas/Search/UnitTests/Index/IndexDescribeCommandTests.cs

```diff
@@ -18,6 +18,7 @@
 
 namespace AzureMcp.Tests.Areas.Search.UnitTests.Index;
 
+[Trait("Area", "Search")]
```

Comment:
super nit; I wonder if enum can be used for areas? given there a few and we repeat the same string values in multiple places -

```C#
namespace AzureMcp.Tests.Areas;

internal enum Area
{
    Search,
    Storage,
    Core,
    ...
}

```

```c#
[Trait("Area", nameof(Area.Search))]
public class IndexDescribeCommandTests {
...
```

disregard if you think this does not add significant value.